aptoide-ui/src/main/java/cm/aptoide/pt/aptoide_ui/icons/AptoideTVIcon.kt

```diff
@@ -11,14 +11,14 @@ import androidx.compose.ui.graphics.vector.path
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import cm.aptoide.pt.theme.greyLight
-import cm.aptoide.pt.theme.pinkishOrange
+import cm.aptoide.pt.theme.aptoideTVIconOrange
 import cm.aptoide.pt.theme.iconsBlack
 
 @Preview
 @Composable
 fun TestAptoideTVIcon() {
   Image(
-    imageVector = getAptoideTVIcon(pinkishOrange, iconsBlack, greyLight),
+    imageVector = getAptoideTVIcon(aptoideTVIconOrange, iconsBlack, greyLight),
```

Comment:
The color is changed only on the preview. Also set it in the Theme file to take change in the actual app.